src/index.js

```diff
@@ -19,7 +19,9 @@
 
 import "cockpit-dark-theme";
 import React from 'react';
+
```

Comment:
This newline is a bit weird, as react and react-dom really belong together. Possibly that needs to go into the config, too?